File: Programmers/Lv0/합성수찾기/Solution.java
package Lv0.합성수찾기

class Solution {

    private boolean isComposition(int n) {
        int cnt = 0;

        for (int i = 1; i <= n; i++) {
            if (n % i == 0) cnt++;
        }

        return cnt >= 3;
    }

    public int solution(int n) {
        int answer = 0;

        for (int i = 4; i <= n; i++) {
            if (isComposition(i)) answer++;
        }

        return answer;
    }
}

File: Programmers/Lv0/합성수찾기/solution.py
def is_composition(number: int) -> bool:
    cnt = 0

    for i in range(1, number + 1):
        if number % i == 0:
            cnt += 1

    return cnt >= 3


def solution(n):
    answer = 0

    for i in range(4, n + 1):
        if is_composition(i):
            answer += 1

    return answer
